ui/components/app/confirm-page-container/confirm-page-container-content/confirm-page-container-content.component.js

```diff
@@ -160,11 +174,76 @@ export default class ConfirmPageContainerContent extends Component {
         {this.renderContent()}
         {!supportsEIP1559V2 &&
```

Comment:
I know this isn't your addition but I'm wondering on this do we not show other types of errors as in not `INSUFFICIENT_FUNDS_ERROR_KEY`, when `supportsEIP1559V2 === true`? cc @jpuri @danjm

Reply:
done.